src/Task17/task17.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+function outputArray (int $n): array
+{
+    $a = [];
+    do
+    {
+        array_push($a, $n);
```

Comment:
Use $a[] = $n instead of array_push() here, because array_push() makes overhead of function call.